### PR TITLE
Add a configuration for creating analytics dashboards for users.

### PR DESCRIPTION
**Pull request by ShalkiWenushika**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by tgtshanika — APPROVED** [review]

File: en/docs/learn/analytics/customizing-analytics-dashboards.md [hunk 1]
```diff
@@ -2,8 +2,24 @@
 
 WSO2 API Manager Analytics has five different dashboards that are namely, API Analytics, Application Analytics, Business Analytics, Monitoring, and Reports.
 
-The users are not be allowed to modify the default dashboards such as, either to modify the layout of the widget or add custom widgets to a particular dashboard. If you need to modify one of the default dashboards, you need to make a copy of the dashboard and the do the modifications to the copy of the dashboard as described below.
+The users are not be allowed to modify the default dashboards such as, either to modify the layout of the widget or add custom widgets to a particular dashboard. If you need to modify one of the default dashboards, you need to make a copy of the dashboard and the do the modifications to the copy of the dashboard as described in step 1 to 5.
```
Comment (tgtshanika) [inline en/docs/learn/analytics/customizing-analytics-dashboards.md:5]: The users are not allowed

Comment (fazlan-nazeem) [inline en/docs/learn/analytics/customizing-analytics-dashboards.md:5]:
```suggestion
The users are not allowed to modify the default dashboards - i.e modify the layout of the widget or add custom widgets to a particular dashboard. If you need to modify one of the default dashboards, you need to make a copy of the dashboard and do modifications to the copy of the dashboard as described in steps 1 to 5.
```

**Comment by fazlan-nazeem** [comment]
To avoid grammatical mistakes when creating doc PRs,  you can use the Grammarly extension.

File: en/docs/learn/analytics/customizing-analytics-dashboards.md [hunk 2]
```diff
@@ -2,8 +2,24 @@
...
 
+In order to make it possible for other users to create dashboards, you need to append underscore parts to existing scopes in `deployment.yaml` file resides in `<Analytics_HOME>/conf/dashboard` directory.
```
Comment (fazlan-nazeem) [inline en/docs/learn/analytics/customizing-analytics-dashboards.md:7]:
```suggestion
In order to make it possible for other users to create dashboards, you need to append "_<tenant domain>" to existing scopes in the `deployment.yaml` file which resides in `<Analytics_HOME>/conf/dashboard` directory. 
```

**Review by tgtshanika — APPROVED** [review]

**Pull request merged**